versions.txt

```diff
@@ -15,7 +15,7 @@ operator-opamp-bridge=0.103.0
 
 # Represents the current release of Java instrumentation.
 # Should match autoinstrumentation/java/version.txt
```

Comment:
it is already 2.5.0 here: https://github.com/open-telemetry/opentelemetry-operator/blob/main/autoinstrumentation/java/version.txt

Reply:
We are intentionally pinning 1.x due to breaking change on the semantic conventions

Reply:
right. is there any info about this and the plan for upgrade somewhere?

the comment should probably be updated in any case?

Reply:
It should be documented in release.yaml. 

Users can override the image in the instrumentation CR

Reply:
aha, yes I can see that there is a comment about it in RELEASE.md. might help others if it were a bit easier to find though.

should I just close this PR then?

Reply:
There is also this pinned issue: https://github.com/open-telemetry/opentelemetry-operator/issues/2542

Reply:
@tormodatt closing is appropriate as we are not ready to accept making 2.x the default image.